public/API/API_GetGameExtended.php

```diff
@@ -50,28 +50,99 @@
  *    string    Expiration                date the claim will expire
  */
 
+use App\Community\Models\AchievementSetClaim;
 use App\Platform\Enums\AchievementFlag;
+use App\Platform\Models\Achievement;
+use App\Platform\Models\Game;
+use Carbon\Carbon;
 
 $gameID = (int) request()->query('i');
 $flag = (int) request()->query('f', (string) AchievementFlag::OfficialCore);
-getGameMetadata($gameID, null, $achData, $gameData, flag: $flag, metrics: true);
 
-if ($gameData === null) {
+if ($game = Game::with('system')->find($gameID)) {
+    $gameSetClaims = AchievementSetClaim::find($gameID);
+}
+
+if ($game) {
+    $gameData =
+        [
+            'ID' => $game->ID,
+            'Title' => $game->Title,
+            'ConsoleID' => $game->ConsoleID,
+            'ForumTopicID' => $game->ForumTopicID,
+            'Flags' => null, // Always '0', this is different in the extended endpoint test for some reason
+            'ImageIcon' => $game->ImageIcon,
+            'ImageTitle' => $game->ImageTitle,
+            'ImageIngame' => $game->ImageIngame,
+            'ImageBoxArt' => $game->ImageBoxArt,
+            'Publisher' => $game->Publisher,
+            'Developer' => $game->Developer,
+            'Genre' => $game->Genre,
+            'Released' => $game->Released,
+            'IsFinal' => $game->IsFinal,
+            'RichPresencePatch' => md5($game->RichPresencePatch),
+            'GuideURL' => $game->GuideURL,
+            'Updated' => $game->Updated,
+        ];
```

Comment:
To match indentation:
```suggestion
    $gameData = [
        'ID' => $game->ID,
        'Title' => $game->Title,
        'ConsoleID' => $game->ConsoleID,
        'ForumTopicID' => $game->ForumTopicID,
        'Flags' => null, // Always '0', this is different in the extended endpoint test for some reason
        'ImageIcon' => $game->ImageIcon,
        'ImageTitle' => $game->ImageTitle,
        'ImageIngame' => $game->ImageIngame,
        'ImageBoxArt' => $game->ImageBoxArt,
        'Publisher' => $game->Publisher,
        'Developer' => $game->Developer,
        'Genre' => $game->Genre,
        'Released' => $game->Released,
        'IsFinal' => $game->IsFinal,
        'RichPresencePatch' => md5($game->RichPresencePatch),
        'GuideURL' => $game->GuideURL,
        'Updated' => $game->Updated,
    ];
```